<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>973</width>
    <height>628</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>973</width>
    <height>628</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>973</width>
    <height>628</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: rgb(243, 243, 243);</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>621</width>
     <height>551</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QVideoWidget" name="video_widget" native="true">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(38, 38, 38);</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>560</y>
     <width>621</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="playButton">
      <property name="minimumSize">
       <size>
        <width>41</width>
        <height>41</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>41</width>
        <height>41</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/newPrefix/48-483851_play-button-png-icono-gmail-png-clipart.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="timelineSlider">
      <property name="maximumSize">
       <size>
        <width>851</width>
        <height>22</height>
       </size>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="timer">
      <property name="minimumSize">
       <size>
        <width>41</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>41</width>
        <height>31</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 9pt &quot;Georgia&quot;;</string>
      </property>
      <property name="text">
       <string>00:00</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="goToPlayer">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>570</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>301</width>
     <height>31</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>301</width>
     <height>31</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;Georgia&quot;;
background-color: rgb(212, 212, 212);</string>
   </property>
   <property name="text">
    <string>Вернуться в плеер</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>530</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>301</width>
     <height>31</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>301</width>
     <height>31</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;Georgia&quot;;
background-color: rgb(212, 212, 212);</string>
   </property>
   <property name="text">
    <string>Сохранить файл</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loadFileButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;Georgia&quot;;
background-color: rgb(212, 212, 212);</string>
   </property>
   <property name="text">
    <string>Загрузить файлы</string>
   </property>
  </widget>
  <widget class="QComboBox" name="select_option">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>299</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>299</width>
     <height>31</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;Georgia&quot;;
background-color: rgb(221, 221, 221);</string>
   </property>
   <item>
    <property name="text">
     <string>Вырезать фрагмент</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Склеить несколько файлов в один</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Вырезать аудиодорожку из файла</string>
    </property>
   </item>
  </widget>
  <widget class="QListWidget" name="files_list">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>301</width>
     <height>101</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>301</width>
     <height>101</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>301</width>
     <height>101</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(218, 218, 218);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>301</width>
     <height>31</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>301</width>
     <height>31</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 14pt &quot;Georgia&quot;;
background-color: rgb(186, 186, 186);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Загруженные файлы:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="applyButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;Georgia&quot;;
background-color: rgb(212, 212, 212);</string>
   </property>
   <property name="text">
    <string>Применить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>280</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>41</width>
     <height>41</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>41</width>
     <height>41</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/newPrefix/84c1b4d37815ca0b78c2e2a3bbd5555e.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteFileButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>90</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;Georgia&quot;;
background-color: rgb(212, 212, 212);</string>
   </property>
   <property name="text">
    <string>Удалить файл</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVideoWidget</class>
   <extends>QWidget</extends>
   <header>PyQt5.QtMultimediaWidgets</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../imgs.qrc"/>
 </resources>
 <connections/>
</ui>
